docs/admin/install.rst

```diff
@@ -650,6 +650,23 @@ After enabling it, you should set :setting:`ENABLE_HTTPS` in the settings:
    :setting:`ENABLE_HTTPS`,
    :ref:`production-site`
 
+
+Set properly SECURE_HSTS_SECONDS
+++++++++++++++++++++++++++++++++
+
+Set this value to 0 so as to make sure that everything works as expected,
+otherwise you could prevent the client from visiting the site.
```

Comment:
Sorry, but this is nonsense. In most cases it should be set to reasonably high value to enforce HSTS and bring increased security. It should be set to 0 only in case HTTP is also supposed to be used on the site (what I hope nobody seriously considers these days).

Reply:
@nijel , Yes you're right that it should be set to a high value so as to prevent a man-in-the-middle attack. Its set to 0 to allow HTTP and may be set to a lower value for testing purposes ensuring all assets are servers securely.
I apologize for not reading and understanding it completely.

Reply:
@nijel - If we set the value to 31536000 seconds, i.e. 1 year(As per Django documentation) is fine or I have to explain all scenarios in the document.